src/game/rdr/data/ObjModels.hpp

```diff
@@ -0,0 +1,322 @@
+#pragma once
+#include <string>
+#include <unordered_set>
+
+
+namespace YimMenu::Data
+{
+	struct ObjModelInfo
+	{
+		std::string model;
+		std::string type;
```

Comment:
I think this should be named `name` instead of `type`

Reply:
Plans for this, type will be changed to categories and name will be added later on in a future update down the road